drivers/console/bm_serial.c

```diff
@@ -14,14 +14,13 @@
 #include <logging/log.h>
```

Comment:
^ Comments should be updated to remove mention of UART Pipe and add a description for bm_serial. Probably should add Sofar to the copyright - this driver has substantially diverged from the original uart pipe code.